src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '/src/styles/blocks/variables';
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__total {
+    display: flex;
+  }
+
+  &__number-of-reviews {
+    color: rgba(6, 11, 53, 1);
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__total--n div:nth-child(-n + 4) {
```

Comment:
The selector `.stars__total--n` seems to be incorrect. It looks like you're trying to select a modifier of the `stars__total` block, but the `--n` part doesn't seem to follow the BEM naming convention. Please review this part of your code and make sure the class names are correct.